### DISCUSSION TITLE
Question about new signups

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#382] [Q&A] [answered]
Is there any way to know to what ecoverse a new signup belongs? Every now end then I look through the list of applicants and see names I don't recognise. Right now there is no way of telling if they have applied for the Digicampus ecoverse, if they're Alkemio team members or just passers-by who have found the hub via other ways.  

How will it work if we have multiple ecoverses on the hub?

**Comment by techsmyth** [comment, marked as answer]
Good to be getting this clear, something we need to explain better.

The current flow is as follows:
- A user signs up to the platform, creating a profile etc
- The user then applies to join a particular ecoverse,  they are not part of *any* ecoverse by default
- 
So if the user wants to join multiple ecoverses then they need to submit multiple applications

This flow may be different when we bring in invitations.

Help?

**Reply by techsmyth** [reply]
What would I think be useful is if you can easier see more information about the user that has applied on the applicants page e.g. the same popup that is shown in the search page so that you can see where they are members. useful?

**Reply by FloriSpoelstra** [reply]
Ah yes I think I get it now. So once someone has signed up they can apply for an ecoverse directy after that. That does raise another question: can someone - after signup - apply directly for a challenge and/or opportunity? Or is it a layered approach: they first apply for the ecoverse, get accepted, then apply for a challenge, get accepted and then apply for an opportunity. 

From a user perspective I think it would be great if you can apply directly for an ecoverses/challenge/opportunity as part of the sign-up proces. Or we need to remind people somehow that they still have to apply for something to become part of a community. Because that is the actual problem: most people are not finding their way to the apply buttons (we have far more sign-ups than we have applications).

And yes, bringing in invitations will fix it. I don't think extra info about a user will help with this specific issue.

**Reply by Comoque1** [reply]
Hello Flori, regarding your question - the current behavior of user application is layer based: apply for ecoverse, approve > apply for challenge, approve. Creating applications on opportunity is not enabled on the client for the moment (currently possible using /graphql).

**Reply by techsmyth** [reply]
As a short term workaround, you can also go in via the graphql query (assuming you are logged in):

query {
  ecoverse(ID: "digicampus") {
    nameID
    community {
      applications {
        id
        user {
          nameID
        }
        lifecycle {
          state
        }
      }
    }
    challenges {
      nameID
      community {
      applications {
        id
        user {
          nameID
        }
        lifecycle {
          state
        }
      }
    }
    }
  }
}

**Comment by FloriSpoelstra** [comment]
I will close this one. It is now clear to me how the sign-up process is designed. It did raise a new issue, which I have posted seperately [assisting new members on platform](/alkem-io/alkemio/discussions/396).
